app/controllers/sign_up/completions_controller.rb

```diff
@@ -0,0 +1,24 @@
+module SignUp
+  class CompletionsController < ApplicationController
```

Comment:
That might work for now, but if we decide that previously proofed users can lose their proofing status, then we'll also need to check to see if they are proofed or not. So, maybe the condition is they have to be either fully signed up or proofed?